src/components/AppNavigation/CalendarList/CalendarListNew.vue

```diff
@@ -22,94 +22,250 @@
 <template>
 	<AppNavigationItem
 		v-if="!showForm"
-		icon="icon-add"
-		:class="{disabled: disabled}"
-		:title="$t('calendar', 'New calendar')"
-		@click.prevent.stop="openDialog" />
-
-	<ActionInput
-		v-else
-		v-click-outside="closeNewCalendarForm"
-		:icon="inputIcon"
-		:value="name"
-		:disabled="isCreating"
-		@submit.prevent.stop="addCalendar">
-		{{ $t('calendar', 'Name of calendar') }}
-	</ActionInput>
+		class="app-navigation-entry-new-calendar"
+		:class="{'app-navigation-entry-new-calendar--open': isOpen}"
+		:title="$t('calendar', '+ New calendar')"
+		@click.prevent.stop="openDialog">
+		<template slot="counter">
```

Comment:
Please use the actions slot https://nextcloud-vue-components.netlify.com/#/Components/App%20containers?id=appnavigationitem
![image](https://user-images.githubusercontent.com/14975046/67483045-30a8a800-f665-11e9-89b8-8d45b7f86be6.png)

Reply:
You can't replace the icon when you use actions.  And i would like to have the icon-add, not the three dots

Reply:
Then fix it, don't abuse the current components please

Reply:
https://github.com/nextcloud/nextcloud-vue/pull/686